### PR TITLE
Homework week2

### PR DESCRIPTION
**Pull request by ThreePalmTrees**
Hi, 
Sorry for handing the homework late, I was a little bit busy with home _(Syria)_

### CODE REVIEW

File: Week2/homework/maartjes_work.js [hunk 1]
```diff
@@ -1,47 +1,54 @@
...
     duration: 40
   }
 ];
 
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+let hourSalary = 25;
+const earnings = tasks
+  .map(x => x.duration / 60) // transforming all task durations to hours
```
Comment (silksil) [inline Week2/homework/maartjes_work.js]: I see you have put comments in there to explain what is happening. Nonetheless, you have not made the variables descriptive. I would advice you to make the variables descriptive, and to minimize the amount of comments in there. Additionally, aim to have a variable name that  is that descriptive that no comments are required . It requires less reading and there is less code you have to maintain. Example: tasks.map(minutesToHours => minutesToHours.duration / 60)

File: Week2/homework/maartjes_work.js [hunk 2]
```diff
@@ -1,47 +1,54 @@
...
+  .filter(y => y >= 2) // filtering and keeping only the tasks that take more than two hours
```
Comment (silksil) [inline Week2/homework/maartjes_work.js]: comment above applies here too.

File: Week2/homework/maartjes_work.js (hunk 1 above)
Comment (ThreePalmTrees) [reply Week2/homework/maartjes_work.js]: Thank you Sil for your feedback,
I've made the variables more descriptive, but kept the comments cause they help me understand what's going on. if that's okay of course.

**Comment by rohni** [comment]
Nice job, and I see you have improved the readability of your variables.

Everything works, and is readable.

**Pull request closed without merging**